src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(FigureColor.values().length);
```

Comment:
Don't create a new instance of Random every time the method is called. It's better to have a single instance as a class member and reuse it. This will be more efficient and will produce a better random sequence.